src/commonTest/kotlin/fr/acinq/lightning/payment/PaymentRequestTestsCommon.kt

```diff
@@ -437,6 +437,19 @@ class PaymentRequestTestsCommon : LightningTestSuite() {
         assertEquals(Features(Feature.VariableLengthOnion to FeatureSupport.Mandatory, Feature.PaymentSecret to FeatureSupport.Mandatory), Features(pr.features))
     }
 
+    @Test
+    fun `filter non-invoice features when parsing invoices`() {
+        // The following invoice has feature bit 20 activated (option_anchor_outputs) without feature bit 12 (option_static_remotekey).
+        // This doesn't satisfy the feature dependency graph, but since those aren't invoice features, we should ignore it.
+        val invoice = "lntb250u1p3jxdmwpp5mnjuf5zwkndq4e29lmvqen8cnwc57aqr0658jlfl3q42667q3lgsdpqdehkuttfdemx76trv5sxvetpw36hyetncqpxsp5a0epznfadhwjrtsrqykjpxu97800grzzjv3kuq3xnlsdgaytqn3s9pqzqqqqqqzqqqqqqqqqqqqqqqqqqqqqsgqk243r3nnm92d8qaam7hgwc5yhupq4wvmnufyyt6kk6hww98a4vtrzxy0pqrpj7dyffykdfx326k0ggcw4l2d08xkknes4unt6vu3ptgqk9zn8r"
```

Comment:
I see that you are checking whether I am paying attention 😄 

Your invoice didn't have bit 20 activated, this one does:

```suggestion
        val invoice = "lntb250u1p3jxdmwpp5mnjuf5zwkndq4e29lmvqen8cnwc57aqr0658jlfl3q42667q3lgsdpqdehkuttfdemx76trv5sxvetpw36hyetncqpxsp5a0epznfadhwjrtsrqykjpxu97800grzzjv3kuq3xnlsdgaytqn3s9pqzqqqqqqzqqqqqqqqqqqqqqqqqqqpqsgqk243r3nnm92d8qaam7hgwc5yhupq4wvmnufyyt6kk6hww98a4vtrzxy0pqrpj7dyffykdfx326k0ggcw4l2d08xkknes4unt6vu3ptgqxesz4j"
```

Reply:
Damn it, I changed it at some point for also include the mandatory unknown feature bit and erased bit 20 :face_with_head_bandage: 

This shows I really need to do some work to instead have a test utility that lets us create invoices with invalid feature bits, I'll improve that.

Reply:
Done in https://github.com/ACINQ/lightning-kmp/pull/367/commits/49928390d38c8cee62abb458fae2e73a1ce5b9ea